analyses/cell-type-wilms-tumor-06/notebook/README.md

```diff
@@ -26,4 +26,12 @@ In brief, we used `Azimuth` to transfer labels from the `Azimuth` fetal full ref
 In brief, we used `Azimuth` to transfer labels from the fetal kidney reference (Stewart et al.) 
 
 - `03_clustering_exploration_{sample-id}.html` is the output of the [`03_clustering_exploration.Rmd`](../notebook_template/03_clustering_exploration.Rmd) notebook template.
-In brief, we explore the clustering results, we look into some marker genes, pathways enrichment and label transfer.
\ No newline at end of file
+In brief, we explore the clustering results, we look into some marker genes, pathways enrichment and label transfer.
+
+
+## Global analysis
+
+At some step during the analysis, a global picture was required to interprete correctly the results and take the best decision for the next steps. 
+
+- `04_annotation_Across_Samples_exploration.html` is the output of the [`04_annotation_Across_Samples_exploration.Rmd`](../notebook/04_annotation_Across_Samples_exploration.Rmd) notebook. 
+In brief, we explored the label transfer results across all samples in the Wilms tumor dataset SCPCP000006.
```

Comment:
I'm suggesting a small text change here to add a bit more context about why we are exploring these results:

```suggestion
The next step in analysis is to identify tumor vs. normal cells.

- `04_annotation_Across_Samples_exploration.html` is the output of the [`04_annotation_Across_Samples_exploration.Rmd`](../notebook/04_annotation_Across_Samples_exploration.Rmd) notebook. 
In brief, we explored the label transfer results across all samples in the Wilms tumor dataset SCPCP000006 in order to identify a few samples that we can begin next analysis steps with.
```